content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
+otherwise specified.
+
+## Purpose and Vision ##
+
+The undergraduate committee aims to create a safe environment for cis
+and trans women as well as non-binary individuals in computer science,
+and to advocate for the inclusion of these marginalized groups in the
+computer science community at large.
+
+## Representation ##
+
+The Women in Computer Science Undergraduate Subcommittee aims to
+represent and support undergraduate cis and trans women, and non-binary
+individuals in computer science. Additionally, we seek to include and
+support members of all underrepresented groups in the fields of
+technology and computing, with the most varied and diverse backgrounds
+possible.
+
+## Values ##
+
+1.  Inclusiveness: The undergraduate commitee is committed to creating
+    and inclusive environment in which women of all backgrounds will
+    feel welcome.
+
+2.  Humility: In all it's activities, the committee will acknowledge and
+    respect the perspectives of those with experiences different to
+    ours, and will strive to understand and incorporate these
+    perspectives.
```

Comment:
There's a typo: it's -> its

I hate using "all" because it's really strong wording. I suggest axing the first four words.